E-Commerce/readme.md

```diff
@@ -46,16 +46,87 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+
+```sql
+SELECT * FROM products WHERE product_name ILIKE '%socks%';
+```
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+```sql
+SELECT p.id, p.product_name, pa.unit_price, PA.SUPP_ID
+    FROM products p INNER JOIN
+    product_availability pa on (p.id = pa.prod_id)
+    WHERE pa.unit_price > 100;
+```
```

Comment:
CORRECT -  stylistically, try and keep text case consistent (i.e. pa.supp_id here, instead of PA.SUPP_ID), although it is irrelevant for correct execution